quick-start.md

```diff
@@ -79,13 +81,13 @@ func main() {
 
 	err = df.Writer().Mode("overwrite").
 		Format("parquet").
-		Save(ctx, "file:///tmp/spark-connect-write-example-output.parquet")
+		Save(ctx, *filedir)
```

Comment:
```
path := fmt.Sprintf("file://%s/spark-connect-write-example-output.parquet")
Save(ctx, path)
```

Reply:
updated
and this is what it looks like on my local laptop

<img width="385" alt="img" src="https://github.com/user-attachments/assets/f9d0ee88-03a2-4451-8a58-3bbb25e8eaef">

Reply:
Ok, interesting :)